BMM.Api/Implementation/Models/TopSong.cs

```diff
@@ -0,0 +1,8 @@
+namespace BMM.Api.Implementation.Models
+{
+    public class TopSong
```

Comment:
Is TrackWithCount a better name?

Reply:
I don't like this "With..." convention, because it's not extensible in case you want to add something in the future (or you need to name it "SomethingWithSomethingWithSomething") ;)

Reply:
ok. let's leave it